packages/ember-data/tests/integration/setup-container-test.js

```diff
@@ -63,7 +67,7 @@ test("the deprecated adapter:_rest is resolved as adapter:rest", function() {
     deprecated = container.lookup('adapter:_rest');
   });
 
-  ok(deprecated === valid, "they should resolve to the same thing");
+  ok(deprecated._debugContainerKey === valid._debugContainerKey, "they should resolve to the same thing");
```

Comment:
Seems odd to compare the key like this, given that it will always just be the value that you passed to `container.lookup` above (making the test not useful).